tests/integration_tests/security/test_vulnerabilities.py

```diff
@@ -180,6 +180,7 @@ def test_spectre_meltdown_checker_on_host(spectre_meltdown_checker):
         comparator=set_did_not_grow_comparator(
             spectre_meltdown_reported_vulnerablities
         ),
+        ignore_return_code=True,
```

Comment:
is this a temporary fix till we find why checker returns 2?

Reply:
@zulinx86 do you have context around this?

Reply:
I lack the context. In the A/B version of the test we ignore the return codes, so it would make sense to do it here too. I agree we need to figure out if ignoring is the right thing to do, but it could be done as a follow up, while we fix the pipelines.

Reply:
What Pablo says is right. In A/B testing, the test would like just to test whether there is difference between A and B. The result of spectre/meltdown checker can fail due to AMI update. Not to be blocked by this type of failure, we introduced A/B testing. So we should ignore the return code here. But let me check whether it ignores the return code if it is not a PR.

Reply:
I've caught up the discussion here now. Before introduction of A/B testing, this test was run as part of optional PR since it can fail due to AMI updates. My suggestion is to run this test mandatorily on PR but run this separately on nightly test (e.g. run in `.buildkite/pipeline_security.py`) to distinguish from functional test failures.

Reply:
Ok, so IIUC it makes sense to merge this PR to unblock the nightly pipeline and then we can do the suggested change on a follow up?